MARC.js

```diff
@@ -384,16 +404,9 @@ record.prototype.translate = function(item) {
 						.replace(	//chop off any translations, since they may have repeated $e fields
 							new RegExp('\\' + subfieldDelimiter + 'd.+'), '');
 		title = this.extractSubfields(title, '200');
-		item.title = title.a;
-		if(title.e) {
-			//If the title proper did not end in a punctuation mark, we should add a colon
-			if(item.title.search(/[A-Za-z0-9]\s*/) != -1) {
-				item.title += ':';
-			}
+		item.title = glueTogether(title.a, title.e, ': ');
+		item.title = clean(item.title);
```

Comment:
Okay, I guess the simpliest would be to add in the `clean` function a line (first):
```
if (value == null) return null;
```
Is that okay?